electron/src/updater/updateservice.ts

```diff
@@ -82,15 +85,27 @@ export class UpdateService extends EventEmitter {
 
                 if (targetVersion) {
                     if (criticalPrompt) {
+                        Telemetry.tickEvent("electron.update.available", {
+                            initial: "true",
```

Comment:
It doesn't look like it according to [appInsights docs](https://docs.microsoft.com/en-us/azure/application-insights/app-insights-api-custom-events-metrics#properties)